### PR TITLE
Feature/UI-10 migrate Pill

### PR DESCRIPTION
**Pull request by PiotrGrzechnik**

### CODE REVIEW

File: packages/pill/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- `prop-types: ^15.6.1`
+- `react: ^16.2.0`
+- `react-dom: ^16.2.0`
+
+These packages are required by `@talixo/pill`, but you have to install them manually,
+to avoid having different versions of these in your application.
+
+## Supported props
+
+Additionally, it handles some differently:
```
Comment (rangoo94) [inline packages/pill/README.md]: Something missing there?

File: packages/pill/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Property name | Type      | Default | Description                    
+--------------|-----------|:-------:|--------------------------------
+className     | string    | n/a     | additional class name passed to wrapper
+color         | string    | n/a     | color of text
+style         | object    | n/a     | object with styles
```
Comment (rangoo94) [inline packages/pill/README.md]: Style is just passed with other components

File: packages/pill/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+variant       | string    | n/a     | different variants
```
Comment (rangoo94) [inline packages/pill/README.md]: Does it have any different variants?

File: packages/pill/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+to avoid having different versions of these in your application.
+
+## Supported props
+
+Additionally, it handles some differently:
+
+Property name | Type      | Default | Description                    
+--------------|-----------|:-------:|--------------------------------
+className     | string    | n/a     | additional class name passed to wrapper
+color         | string    | n/a     | color of text
```
Comment (rangoo94) [inline packages/pill/README.md]: Color should be renamed, also - it's not color of text, but the whole pill style

File: packages/pill/tests/Pill.test.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import Pill from '../src/Pill'
+import { shallow } from 'enzyme'
+
+describe('<Pill />', () => {
+  it('renders children correctly', () => {
```
Comment (rangoo94) [inline packages/pill/tests/Pill.test.js]: It's worth to add tests which will include checking if props are passed properly

File: packages/pill/README.md (hunk 3 above)
Comment (PiotrGrzechnik) [reply packages/pill/README.md]: We have a 'ghost mode' like in a button component.

**Pull request merged**